framework/test/unit/components/storage/entities/round.js

```diff
@@ -224,11 +224,11 @@ describe('Round', () => {
 
 	describe('_getResults()', () => {
 		beforeEach(async () => {
-			await storage.entities.Round.create(validRound);
+			return storage.entities.Round.create(validRound);
```

Comment:
How is it effecting any change? If both have different behavior then we have to look carefully on other places as well.

Reply:
This is just for convenience. When you return a Promise in an async function it's gonna be chained and awaited as well.